can_bus/quadruna/BMS/BMS_alerts.json

```diff
@@ -6,113 +6,113 @@
             "id" :107,
-            "description" : "Example"
+            "description" : "CAN TX queue has overflowed."
         },
         "RxOverflow" : {
             "id" :108,
-            "description" : "Example"
+            "description" : "CAN RX queue has overflowed."
+        },
+        "OpenWireCheckFault": {
+            "id" :165,
+            "description" : "Open wire cell detected (cell voltage tap is open circuit)."
+        },
+        "OpenWireCheck_Segment0_GND": {
+            "id" :166,
+            "description" : "Open wire cell detected in segment 0 (cell voltage tap is open circuit)."
+        },
+        "OpenWireCheck_Segment1_GND": {
+            "id" :167,
+            "description" : "Open wire cell detected in segment 1 (cell voltage tap is open circuit)."
+        },
+        "OpenWireCheck_Segment2_GND": {
+            "id" :168,
+            "description" : "Open wire cell detected in segment 2 (cell voltage tap is open circuit)."
+        },
+        "OpenWireCheck_Segment3_GND": {
+            "id" :169,
+            "description" : "Open wire cell detected in segment 3 (cell voltage tap is open circuit)."
+        },
+        "OpenWireCheck_Segment4_GND": {
+            "id" :170,
+            "description" : "Open wire cell detected in segment 4 (cell voltage tap is open circuit)."
         }
     },
     "faults": {
         "MissingVCHeartbeat":{
             "id" :151,
-            "description" : "Example"
+            "description" : "Missing heartbeat signal from the VC."
         },
         "MissingRSMHeartbeat":{
             "id" :152,
-            "description" : "Example"
+            "description" : "Missing heartbeat signal from the RSM."
         },
         "StateMachine" :{
             "id" :154,
-            "description" : "Example"
+            "description" : "BMS has transitioned to fault state."
         },
         "CellUndervoltage": {
             "id" :155,
-            "description" : "Example"
+            "description" : "Cell is critically undervoltage (<3.0V)"
         },
         "CellOvervoltage": {
             "id" :156,
-            "description" : "Example"
+            "description" : "Cell is critically overvoltage (<4.2V)"
         },
         "ModuleCommunicationError": {
             "id" :157,
-            "description" : "Example"
+            "description" : "Failed to communicate with LTC6813 modules (>3x bad PEC15 error checks)."
         },
         "CellUndertemp": {
             "id" :158,
-            "description" : "Example"
+            "description" : "Cell is critically undervoltage (<-20degC)"
         },
         "CellOvertemp": {
             "id" :159,
-            "description" : "Example"
+            "description" : "Cell is critically undervoltage (>45degC if charging, >60degC if not)."
         },
         "Charger": {
             "id" :160,
-            "description" : "Example"
+            "description" : "TODO @Will"
```

Comment:
This and "ChargerExternalShutdown" seem to be redundant to me, maybe remove this message and just use the other?

Reply:
they are different conditions though right? probably useful to specify which one it is?